### DISCUSSION TITLE
Hello, I can't change the resolution or bitrate when using a Raspberry Pi Camera v3. The parameters entered in mediamtx.yml are not being passed. What am I doing wrong?

### DISCUSSION THREAD

**Discussion by infinitoxxx** [bluenviron/mediamtx#1930] [General]
Hello, I can't change the resolution or bitrate when using a Raspberry Pi Camera v3. The parameters entered in mediamtx.yml are not being passed. What am I doing wrong?

**Comment by aler9** [comment]
duplicate of #1929
